.env.default

```diff
@@ -0,0 +1,13 @@
+# TDA Config vars
+TDA_API_KEY=<tda_api_key_here>@AMER.OAUTHAP
+TDA_CALLBACK_URI=<callback_uri_here>
+TDA_USERNAME=<tda_username_here>
+TDA_PASSWORD=<tda_password_here>
+TDA_SECURITY_QUESTIONS_1=<tda_question_list_here>
+TDA_SECURITY_QUESTIONS_2=<tda_question_list_here>
+TDA_SECURITY_QUESTIONS_3=<tda_question_list_here>
+TDA_SECURITY_QUESTIONS_4=<tda_question_list_here>
+TDA_SECURITY_QUESTION_ANSWERS_1=<tda_question_answer_list_here>
```

Comment:
these should be `TDA_SECURITY_QUESTION_ANSWER_X`

Reply:
Agreed!